### ISSUE TITLE
Sometimes when using RCS it becomes impossible to move using standard movement keys.

### ISSUE THREAD

**Issue by PetMudstone** [unitystation/unitystation#6992]
### Environment
- Game Version: Production 4119
- OS Version: Windows 10

### Detailed description of bug
Two times thus far when playing on Solidarity, movement using keys became impossible when using RCS on shuttles. This both means that both walking and moving using RCS thrusters no longer worked. Unfortunately, I don't know exactly why this happened. 
I am not the only one that has experienced this as Bio#8572 on Discord reported experiencing the same bug. In both our cases dying and coming back to life (respawning for me and cloning for Bio) allowed us to move using keys again.

### Steps to reproduce
Note: I am unsure how to properly reproduce this bug 
1. Fly around with RCS on a shuttle, I guess.

**Comment by Aranclanos** [comment]
cant reproduce

**Comment by Aranclanos** [comment]
tried toggling fast the RCS, turning the ship, moving at the same time, buckling and unbuckling, both on local server and staging and nothing

**Comment by PetMudstone** [comment]
Reproduced on staging 21102918 as client. I did this both on the staging server and as a non-headless client connected to localhost. What I did to reproduce the bug was repeatedly turning RCS on and off while holding down movement keys. I was unable to reproduce as a host so I presume this is client only.

Checking the console after the bug occurred yielded this error:
![image](https://user-images.githubusercontent.com/38266309/139758660-e7ac6908-406b-4f7d-be43-2ced149b431f.png)
